### DISCUSSION TITLE
Tesla Fleet API for TeslaMate

### DISCUSSION THREAD

**Discussion by jheredianet** [tobiasehlert/teslamateapi#292] [General]
I was wondering if this project will be compatible with this: https://github.com/teslamate-org/teslamate/blob/master/website/docs/guides/api.md

**Comment by tobiasehlert** [comment]
@jheredianet, until now we've only used the Owner API for the commands in TeslaMateAPI section.
But since that is going away, we'll probably remove the commands endpoint in near future.

**Reply by jheredianet** [reply]
Thank you @tobiasehlert for clarifying this.
